src/reducers/ledger/index.js

```diff
@@ -26,7 +26,7 @@ const ledgerModalReducer = handleActions({
     [combineActions(sendMoney, addAccessKey, signAndSendTransactions, removeAccessKey, addAccessKeySeedPhrase, deleteRecoveryMethod, setupRecoveryMessage, addLedgerAccessKey, getLedgerPublicKey)]: (state, { ready, meta, type }) => ({
```

Comment:
@marcinbodnar is there any reason why we have to list all these actions explicitly? Seems like we can do following:
1) dispatch action from `Signer` when there is actual signing / public key request sent to Ledger
2) handle that action specifically instead of trying to piggyback on existing actions.

Reply:
@vgrichina This is a good idea. The only thing is that at this moment we want to show a specific message for each tx (`textId` state) - I'm using prefix from `defaultCodesFor` for that, which is handy. But I guess I can check `actionsPending` state to know what type of tx will be signed. Or maybe you have a better idea?

Reply:
@vgrichina I resolved it in #862 (few last commits)